CONTRIBUTING.md

```diff
@@ -516,6 +512,11 @@ To keep everything DRY, add `extends BaseModel` to the class and import it from
 
 You could also run `npx typeorm` since here you're not actually loading any ts files, but because regular `npx typeorm` runs inside of node it import from _.ts_ files, so we run it with `ts-node` and our custom server config (check package.json)
 
+#### Syncing the Schema
+
+* For development environments, [TypeORM will kept the database migrations in sync](https://github.com/typeorm/typeorm/blob/master/README.md) with the definitions in _server/models_ when the server is started. Developers must still create and commit a [migration](#creating-a-migration) for any modifications to the ORM / database schema.
+* For production environments, or if a manually sync becomes necessary, run the `npm run db:sync` command.
```

Comment:
```suggestion
* If a manual sync becomes necessary, run the `npm run db:sync` command.
* For production environments, run the `npm run db:migrate` command.
```
I don't think `db:sync` should be run in production.  According to TypeORM's [migration docs](https://typeorm.io/#/migrations/how-migrations-work) you should use `typeorm migration:run` in production (i.e. our `db:migrate` command).